lib/runTx.js

```diff
@@ -28,7 +28,7 @@ module.exports = function (opts, cb) {
     block = new Block()
   }
 
-  if (new BN(block.header.gasLimit).cmp(new BN(tx.gasLimit)) === -1) {
+  if (new BN(block.header.gasLimit).lt(new BN(tx.gasLimit))) {
```

Comment:
Refactoring ok, would be a bit more readable if switched to a ``gte`` comparison (which also has a slightly different semantics than the error message).